.travis.yml

```diff
@@ -18,11 +18,7 @@ matrix:
     - node_js: "8"
       env: UNIT_TEST=true COVERAGE=true
     - node_js: "8"
-      env: INTEGRATION_TEST=react-percy-storybook PERCY_PROJECT=percy/react-percy-storybook PERCY_TOKEN="$STORYBOOK_PERCY_TOKEN"
-    - node_js: "8"
-      env: INTEGRATION_TEST=react-percy PERCY_PROJECT=percy/react-percy PERCY_TOKEN="$REACT_PERCY_TOKEN"
-    - node_js: "8"
-      env: INTEGRATION_TEST=create-react-app PERCY_PROJECT=percy/create-react-app-integration PERCY_TOKEN="$CREATE_REACT_APP_PERCY_TOKEN"
+      env: INTEGRATION_TEST=react-percy-storybook PERCY_PROJECT=percy/percy-storybook PERCY_TOKEN="$STORYBOOK_PERCY_TOKEN"
```

Comment:
Since there's only this one integration test now, you could probably just simplify this to `INTEGRATION_TEST=true` (with corresponding updates to the integration test script) and set the percy token and project globally in travis.

Reply:
@thomashuston Thanks for mentioning it.  Have giving it some consideration, and I think there's a chance we'll want to add a separate integration test for Storybook's vue support in the not-too-distant future, so I'm going to leave this in place for now.

Reply:
(to make it easier to add Vue next..)